src/main/java/de/berlin/lostberlin/controller/OrderController.java

```diff
@@ -60,6 +61,7 @@ public Order getByOrderNr(@PathVariable (value="order_number") String orderNr){
     public Order updateOrderStatus(@PathVariable (value="order_number") String orderNr, @Valid @RequestBody Order orderProfile){
     Order order = orderRepo.findById(orderNr)
             .orElseThrow(() -> new ResourceNotFoundException("Order", "order_number", orderNr));
+    order.setBusinessId(orderProfile.getBusinessId());
     order.setStatus(orderProfile.getStatus());
```

Comment:
It seems like to much logic in the controller, we should consider adding an OrderService, which will encapsulate all this. We should not return strings as "Success" or "Bad Request" - this should be http responses. 

Please, clean up tests and we refactor controller for proper exception handling later.

Reply:
Thanks! I have refactored it already, will make a PR soon!